specification/scvmm/ScVmm.Management/VirtualMachineInstance.tsp

```diff
@@ -100,13 +101,9 @@ interface VirtualMachineInstances {
   list is ArmResourceListByParent<VirtualMachineInstance>;
   /** The operation to power off (stop) a virtual machine instance. */
   @summary("Implements the operation to stop a virtual machine.")
-  stop is ArmResourceActionNoResponseContentAsync<
+  stop is CustomizedArmResourceActionNoResponseContentAsync<
     VirtualMachineInstance,
-    {
-      /** The content of the action request */
-      @body
-      body?: StopVirtualMachineOptions;
```

Comment:
@kartikverma020 Feel free to take a look at the change and let me know if any concern!

The old patter to set `StopVirtualMachineOptions` as optional is not recommanded and we may have [future linter to limit the usage](https://github.com/microsoft/typespec/issues/5993). 

In this pr we updated the spec to an customized template with optional body operation. This change would only produce more correct client method but without any REST change.